repos/aspnetcore.proj

```diff
@@ -11,6 +11,8 @@
     <BuildCommandArgs>$(BuildCommandArgs) --configuration $(Configuration)</BuildCommandArgs>
     <BuildCommandArgs>$(BuildCommandArgs) --ci</BuildCommandArgs>
     <BuildCommandArgs>$(BuildCommandArgs) -bl</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) -bl</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --arch $(Platform)</BuildCommandArgs>
```

Comment:
I tried using [`TargetArchitecture`](https://github.com/aspnet/AspNetCore/blob/master/docs/BuildFromSource.md#build-properties) but we call `./build.sh` which actually passes `-p:TargetArchitecture=x64` unless we set `--arch`